docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx==4.3.0
 sphinx-material==0.0.35
 m2r2==0.3.2
+clouduniqueid
```

Comment:
can you please add latest version number here?
you can get it from gihub repo latest release or from pypi.org